Add NCI-CRDC TCGA and CPTAC studies to news release (#10910)
cBioPortal/cbioportal: docs/News.md

## July 29, 2024

> *   **Added data** consisting of 12,863 samples from 33 TCGA and 10 CPTAC studies from the GDC as part of the NCI-CRDC initiative. The studies are generated though the [ISB-CGC](https://portal.isb-cgc.org/) BigQuery tables.

Reviewer: Should we expand the acronyms and/or link out, in case users aren't familiar with them?

Reviewer: Also, Ben wants to link out directly to the subpage with BigQuery tables rather than the landing page of the ISB-CGC site.

```suggestion
*   **Added data** consisting of 12,863 samples from 33 TCGA and 10 CPTAC studies from the [Genomic Data Commons (GDC)](https://gdc.cancer.gov) as part of the [Cancer Research Data Commons (NCI-CRDC)](https://datacommons.cancer.gov) initiative. The studies are generated through the [ISB-CGC BigQuery tables](https://bq-search.isb-cgc.org/search?status=current).
```

Author: Thanks for the review. Updated.

> *   **Added data** consisting of 12,863 samples from 33 TCGA and 10 CPTAC studies from the [Genomic Data Commons (GDC)](https://gdc.cancer.gov) as part of the [Cancer Research Data Commons (NCI-CRDC)](https://datacommons.cancer.gov) initiative. The studies are generated through the [ISB-CGC BigQuery tables](https://bq-search.isb-cgc.org/search?status=current).

Reviewer 2: Can we change the last sentence to "More information can be found on our FAQ" and link to this page
https://github.com/cBioPortal/cbioportal/blob/699e939482228c55321afad3fd9ee9c237949cf9/docs/user-guide/faq.md#tcga

Author: Thanks @n1zea144! Updated.

    * [Miscellaneous Neuroepithelial Tumor (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=mnet_tcga_gdc) *184 samples*
    * [Non-Seminomatous Germ Cell Tumor (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=nsgct_tcga_gdc) *156 samples*
    * [Pancreatic Adenocarcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=paad_tcga_gdc) *186 samples*
    * [Papillary Renal Cell Carcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=prcc_tcga_gdc) *292 samples*
    * [Papillary Thyroid Cancer (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=thpa_tcga_gdc) *515 samples*
    * [Pleural Mesothelioma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=plmeso_tcga_gdc) *87 samples*
    * [Prostate Adenocarcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=prad_tcga_gdc) *501 samples*
    * [Rectal Adenocarcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=read_tcga_gdc) *171 samples*
    * [Renal Clear Cell Carcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=ccrcc_tcga_gdc) *537 samples*

>     * [Soft Tissue (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=soft_tissue_tcga_gdc) *264 samples*

Author: @jamesqo, should this also be `Soft Tissue Cancer`?

Reviewer: @rmadupuri Ah yes, good catch!

    * [Stomach Adenocarcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=stad_tcga_gdc) *443 samples*
    * [Thymoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=thym_tcga_gdc) *124 samples*
    * [Uterine Carcinosarcoma/Uterine Malignant Mixed Mullerian Tumor (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=ucs_tcga_gdc) *57 samples*
    * [Uveal Melanoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=um_tcga_gdc) *80 samples*

>     * [Breast (CPTAC, GDC)](https://www.cbioportal.org/study/summary?id=breast_cptac_gdc) *154 samples*

Reviewer: ```suggestion
    * [Breast Cancer (CPTAC, GDC)](https://www.cbioportal.org/study/summary?id=breast_cptac_gdc) *154 samples*
```

Reviewer: ^ Niki had mentioned that he prefers "<body part> Cancer" instead of "<body part>", so I will be updating the names of the studies accordingly. See the discussion in \#data-feedback: https://cbioportal.slack.com/archives/C04PHKPKF2R/p1721693960892619?thread_ts=1721329662.398769&cid=C04PHKPKF2R

>     * [CNS/Brain (CPTAC, GDC)](https://www.cbioportal.org/study/summary?id=brain_cptac_gdc) *74 samples*

Reviewer: ```suggestion
    * [CNS/Brain Cancer (CPTAC, GDC)](https://www.cbioportal.org/study/summary?id=brain_cptac_gdc) *74 samples*
```

    * [Colon Adenocarcinoma (CPTAC, GDC)](https://www.cbioportal.org/study/summary?id=coad_cptac_gdc) *109 samples*
    * [Head and Neck Carcinoma, Other (CPTAC, GDC)](https://www.cbioportal.org/study/summary?id=ohnca_cptac_gdc) *150 samples*
    * [Lung Adenocarcinoma (CPTAC, GDC)](https://www.cbioportal.org/study/summary?id=luad_cptac_gdc) *180 samples*
    * [Lung Squamous Cell Carcinoma (CPTAC, GDC)](https://www.cbioportal.org/study/summary?id=lusc_cptac_gdc) *90 samples*

>     * [Ovary/Fallopian Tube (CPTAC, GDC)](https://www.cbioportal.org/study/summary?id=ovary_cptac_gdc) *113 samples*

Reviewer: ```suggestion
    * [Ovarian Cancer (CPTAC, GDC)](https://www.cbioportal.org/study/summary?id=ovary_cptac_gdc) *113 samples*
```

Reviewer: Not 100% sure about this one since I am removing `Fallopian Tube`, but this sounds smoother than `Ovary/Fallopian Tube Cancer`. Additionally, none of the existing studies we have on the ovary mention the fallopian tube. Thoughts?

Author: That makes sense. Lets have it as `Ovarian Cancer` to be consistent with other studies.

>     * [Pancreas (CPTAC, GDC)](https://www.cbioportal.org/study/summary?id=pancreas_cptac_gdc) *139 samples*

Reviewer: ```suggestion
    * [Pancreatic Cancer (CPTAC, GDC)](https://www.cbioportal.org/study/summary?id=pancreas_cptac_gdc) *139 samples*
```
